.github/workflows/broken_link_check.yml

```diff
@@ -0,0 +1,22 @@
+name: Broken link check
+on:
+  schedule:
+    - cron: "* * * * 1"
+
+jobs:
+  broken_link_checker_job:
+    runs-on: ubuntu-latest
+    name: Check for broken links
+    steps:
+      - name: Check for broken links
+        id: link-report
+        uses: celinekurpershoek/link-checker@v1.0.1
+        with:
+          # Required:
+          url: "https://featuretools.alteryx.com/en/latest/"
+          # optional:
+          honorRobotExclusions: true
+          ignorePatterns: "*github*,*google*,*twitter*,*slack*,*stackoverflow*,*anaconda*"
```

Comment:
Why are we ignoring this?

Reply:
I can put some of these back in but a couple of these produced false positives like the pull requests in release notes.